vcpkg/examples/assetcaching-script.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Asset Caching with an alternate storage source
```

Comment:
I think this example should focus more on being a "How To" or a "Tutorial". The objective of a How To article is to first promise the user where they will end up (with an `x-script` asset cache pointing at NuGet in ADO) and then successfully transport them there (via concrete steps: "First, do X." "Then, do Y.").

(consider browsing https://diataxis.fr/how-to-guides/ for more philosophical inspiration)

```suggestion
title: How to create an x-script Asset Caching source for NuGet
```

Reply:
If you change the title to reflect it being a how-to, you probably should also change the file name.